code/game/gamemodes/cult/runes.dm

```diff
@@ -159,123 +164,121 @@ structure_check() searches for nearby cultist structures required for the invoca
 		return N
 	return 0
 
+//Rite of Binding: Turns a nearby rune and a paper on top of the rune to a talisman, if both are valid.
+/obj/effect/rune/imbue
+	cultist_name = "Create Talisman"
+	cultist_desc = "Transforms papers and valid runes into talismans."
+	invocation = null //no talisman made, no invocation.
+	icon_state = "3"
+	color = rgb(0, 0, 255)
+
+/obj/effect/rune/imbue/invoke(mob/living/user)
+	var/turf/T = get_turf(src)
+	var/list/papers_on_rune = list()
+	var/entered_talisman_name
+	var/obj/item/weapon/paper/talisman/talisman_type
+	var/list/possible_talismans = list()
+	for(var/obj/item/weapon/paper/P in T)
+		if(!P.info)
+			papers_on_rune.Add(P)
+	if(!papers_on_rune.len)
+		user << "<span class='cultitalic'>There must be a blank paper on top of [src]!</span>"
+		fail_invoke()
+		log_game("Talisman Imbue rune failed - no blank papers on rune")
+		return
+	if(rune_in_use)
+		user << "<span class='cultitalic'>[src] can only support one ritual at a time!</span>"
+		fail_invoke()
+		log_game("Talisman Imbue rune failed - more than one user")
+		return
+	var/obj/item/weapon/paper/paper_to_imbue = pick(papers_on_rune)
+	for(var/I in subtypesof(/obj/item/weapon/paper/talisman) - /obj/item/weapon/paper/talisman/malformed - /obj/item/weapon/paper/talisman/supply)
+		var/obj/item/weapon/paper/talisman/J = I
+		var/talisman_cult_name = initial(J.cultist_name)
+		if(talisman_cult_name)
+			possible_talismans[talisman_cult_name] = J //This is to allow the menu to let cultists select talismans by name
+	entered_talisman_name = input(user, "Choose a talisman to imbue.", "Talisman Choices") as null|anything in possible_talismans
+	if(!Adjacent(user) || !src || qdeleted(src) || user.incapacitated())
+		return
+	talisman_type = possible_talismans[entered_talisman_name]
+	user.say("H'drak v'loso, mir'kanas verbot!")
+	visible_message("<span class='warning'>Dark power begins to channel into the paper!.</span>")
+	rune_in_use = 1
+	if(!do_after(user, 100, target = get_turf(user)))
+		rune_in_use = 0
+		return
+	new talisman_type(get_turf(src))
+	visible_message("<span class='warning'>[src] glows with power, and bloody images form themselves on [paper_to_imbue].</span>")
+	qdel(paper_to_imbue)
+	rune_in_use = 0
+
 var/list/teleport_runes = list()
-//Rite of Translocation: Warps the user to a random teleport rune with the same keyword.
 /obj/effect/rune/teleport
 	cultist_name = "Teleport"
-	cultist_desc = "Warps the user to a random rune of the same keyword."
+	cultist_desc = "warps everything above it to another chosen teleport rune."
```

Comment:
Not having a capital is so that the message you get on creating a rune doesn't have random capitals; this is robustin porting one of my changes incompletely.